rancher/v1.6/en/environments/index.md

```diff
@@ -121,6 +121,15 @@ At any time, you can delete the environment templates as they are only used when
 | Secrets | CRUD | CRUD | CRUD | R |
 | Certificates | CRUD | CRUD | CRUD | R |
 | Registries  | CRUD | CRUD | CRUD | R |
+| User Stacks | CRUD | CRUD | CRUD | R
+| Infra Stacks | CRUD | RUD | R | R
```

Comment:
Can you clarify what user and infra stacks are?

Reply:
Sure!